src/vendors/pkce.ts

```diff
@@ -50,7 +50,12 @@ export function generateVerifier(prefix?: string): string {
 export function computeChallenge(str: string): PromiseLike<any> {
   var buffer = new TextEncoder().encode(str);
   return crypto.subtle.digest("SHA-256", buffer).then(function (arrayBuffer) {
-    var hash = String.fromCharCode.apply(null, new Uint8Array(arrayBuffer));
+    // Works in practice because fromCharCode just needs to be able to iterate over the values
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array
+    var hash = String.fromCharCode.apply(
+      null,
+      (new Uint8Array(arrayBuffer) as unknown) as number[]
+    );
```

Comment:
I think the problem isn't `fromCharCode` but `apply`, it's typed to accept an array while it accepts an array-like object, which `Uint8Array` apparently is. Spec: https://tc39.es/ecma262/#sec-function.prototype.apply

Would this be a better solution to avoid assertions and comments?

```suggestion
    var hash = String.fromCharCode.apply(
      null,
      [...new Uint8Array(arrayBuffer)]
    );
```